src/index.ts

```diff
@@ -392,6 +395,76 @@ export function apply(ctx: Context, config: Config) {
               r2: true,
             }
           }
+          case 'comfyui': {
+            const workflowText2Image = config.workflowText2Image ? resolve(ctx.baseDir, config.workflowText2Image) : '../data/default-comfyui-t2i-wf.json'
+            const workflowImage2Image = config.workflowImage2Image ? resolve(ctx.baseDir, config.workflowImage2Image) : '../data/default-comfyui-i2i-wf.json'
+            const workflow = image ? workflowImage2Image : workflowText2Image
+            logger.debug('workflow:', workflow)
+            const prompt= require(workflow)
+
+            // have to upload image to the comfyui server first
+            if (image) {
+              const body = new FormData();
+              const capture = /^data:([\w/-]+);base64,(.*)$/.exec(image.dataUrl)
```

Comment:
```suggestion
              const capture = /^data:([\w/.+-]+);base64,(.*)$/.exec(image.dataUrl)
```

ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types